.github/workflows/basic.yml

```diff
@@ -42,17 +43,34 @@ jobs:
             compiler: {c: gcc, cxx: g++}
             shared_library: 'OFF'
             os_provider: 'OFF'
+          # TODO: Enable sanitizer checks with pool tracking enabled. There's
```

Comment:
The https://github.com/oneapi-src/unified-memory-framework/pull/140 PR enables that